### PR TITLE
secureConnectionStart is mandatory #16

### PR DESCRIPTION
**Pull request by plehegar**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -745,8 +743,7 @@
...
        <a href="#widl-PerformanceResourceTiming-connectStart">connectStart</a> immediately before initiating the connection to the server and record the time as <a href="#widl-PerformanceResourceTiming-connectEnd">connectEnd</a> immediately after the connection to the server or the proxy is established. A user agent may need multiple retries before this time. Once connection is established set the value of <a href="#widl-PerformanceResourceTiming-nextHopProtocol">nextHopProtocol</a> to the ALPN ID used by the connection. If a connection can not be established, abort the remaining steps.
       </li>
-      <li>If the user agent supports the <a href="#widl-PerformanceResourceTiming-secureConnectionStart">secureConnectionStart</a> attribute,
-      in step <a href="#step-connect-start">3.13</a>, a user agent SHOULD also carry out these additional steps:
+      <li>The user agent MUST set the <a href="#widl-PerformanceResourceTiming-secureConnectionStart">secureConnectionStart</a> attribute as follows:
         <ol>
         <li>If the scheme of the current resource is HTTPS, the user agent MUST record the
```
Comment (igrigorik) [inline index.html]: This is unrelated to this pull, but FWIW, I think we need to update "if the scheme is HTTPS" to "when a secure transport is used" to match the earlier update. /cc @toddreifsteck

**Comment by igrigorik** [comment]
lgtm.

**Comment by plehegar** [comment]
@toddreifsteck +1?

**Comment by toddreifsteck** [comment]
+1 confirmed

**Pull request merged**